conf/gulp-tasks/templates.gulpfile.js

```diff
@@ -55,19 +58,15 @@ function precompileTemplates () {
         return declare.processNameByPath(path, '').replace('.', '/');
       }
     }))
-    .pipe(concat('answerstemplates.compiled.min.js'))
+    .pipe(concat('answerstemplates.precompiled.min.js'))
```

Comment:
For my own curiosity, what's this and how is it generated?

Reply:
It's precompiling the templates (adding js wrapper code, compiling the handlebars) and then concatenating all into one file. This is the source file for both of the bundles, so I made it so precompileTemplates only happens once on a `gulp`. It is generated into a file `answerstemplates.precompiled.min.js` in the dist folder that is read by both the bundle types. This is just common js code that is not transpiled into a specific bundle type.

Reply:
Is this something that will appear in the CDN now? Without a specific bundle type, is this an asset someone could actually use? If not, I'm not sure we want it in the CDN.